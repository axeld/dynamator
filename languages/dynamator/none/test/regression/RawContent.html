<html>
<head>
  <title>Raw Content Override</title>
</head>
<body>
<div id="div_3">
<p>fail</p>
</div>
<div class="div_4">
<p><span id="span_1">fail</span></p>
</div>
<div class="x div_4">
<p>fail</p>
<p>fail</p>
</div>
<div class="div_4 x">
<p>fail</p>
<p>fail</p>
</div>
<div class="x div_4 x">
<p>fail</p>
<p>fail</p>
</div>

</body>
</html>
