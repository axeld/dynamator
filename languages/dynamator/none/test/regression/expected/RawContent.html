<html>
  <head>
    <title>Raw Content Override</title>
  </head>
  <body>
    <div id="div_3">
      <p>success!</p>
    </div>
    <div class="div_4">
      <p>success!</p>
    </div>
    <div class="x div_4">
      <p>success!</p>
    </div>
    <div class="div_4 x">
      <p>success!</p>
    </div>
    <div class="x div_4 x">
      <p>success!</p>
    </div>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
